tools/serverpod_cli/lib/src/analyzer/entities/validation/protocol_validator.dart

```diff
@@ -300,6 +282,35 @@ void _collectNodesWithNestedNodesErrors(
   }
 }
 
+dynamic _extractNodeValue(
```

Comment:
If we are using `dynamic` there should really be some comment about what return types we can expect. (Generally avoid `dynamic` where possible.)

Reply:
It has to be dynamic as this is what the Yaml parser is extracting and the source could really be anything.